server/utils/startup.ts

```diff
@@ -20,32 +21,74 @@ export function checkPendingMigrations() {
       line.startsWith("down")
     );
 
-    if (pendingMigrations.length) {
-      Logger.warn("You have pending migrations");
-      Logger.warn(
-        pendingMigrations
-          .map((line, i) => `${i + 1}. ${line.replace("down ", "")}`)
-          .join("\n")
-      );
+    return pendingMigrations;
+  } catch (err) {
+    Logger.warn(chalk.red("Failed to fetch pending migrations!"));
+    if (err.message.includes("ECONNREFUSED")) {
       Logger.warn(
-        "Please run `yarn db:migrate` or `yarn db:migrate --env production-ssl-disabled` to run all pending migrations"
+        chalk.red(
+          `Could not connect to the database. Please check your connection settings.`
+        )
       );
-
-      process.exit(1);
+    } else {
+      Logger.warn(chalk.red(err.message));
     }
+    process.exit(1);
+  }
+}
+
+function runMigrations() {
+  try {
+    Logger.info("database", "Running migrations...");
+    const cmdResult = execSync(
+      `yarn db:migrate${
+        env.PGSSLMODE === "disable" ? " --env=production-ssl-disabled" : ""
+      }`
+    );
+    const cmdOutput = Buffer.from(cmdResult).toString("utf-8");
+    Logger.info("database", cmdOutput);
+    Logger.info("database", "Done.");
   } catch (err) {
+    Logger.warn(chalk.red("Failed to run migrations!"));
```

Comment:
```suggestion
```

This is resulting in a double error, seems fine without?
<img width="647" alt="image" src="https://user-images.githubusercontent.com/380914/231049034-73b0a2df-1b3f-4a13-ad87-6a2d7d4446d2.png">

Reply:
I believe removing `db:migrate` from `Makefile` resolves this. No further action needed?

Reply:
I don't think it changes it to be honest, I'm easy either way here

Reply:
The first error is logged from within the `sequelize` cli :see_no_evil: 

Catching errors centrally within `checkMigrations` now.